src/Ldap.php

```diff
@@ -634,10 +674,55 @@ protected function getAccount($acctname, array $attrs = null)
     }
 
     /**
+     * Selects current parameters on new connections, last when reconnecting.
+     *
+     * @param string $method
+     *   Whether the connect or bind method is the caller.
+     * @param string $parameter
+     *   The parameter name.
+     * @param mixed $property
+     *   The value of the parameter as set in an instance property.
+     * @return mixed
+     *   If a reconnect attempt is being made, the value used for the parameter
+     *   last time it was supplied by an external invocation. Otherwise, the
+     *   value
+     */
+    protected function selectParam($method, $parameter, $property)
+    {
+        if ($this->reconnectCount > 0) {
+            return self::coalesce(
+                $this->lastConnectBindParams[$method][$parameter],
+                $property
+            );
+            return self::coalesce(
```

Comment:
What point am I missing here? Can this return be reached at all?

Reply:
No, thanks for catching this!

I went for the version that includes the isset() check, but all possible values will always be set in this PR anyway (or I'd add another test that covers the unset case!). It becomes relevant when one attempts to merge this with PR#64 to support SASL binds.